proxyclient/m1n1/proxy.py

```diff
@@ -712,7 +712,8 @@ def reboot(self):
     def write64(self, addr, data):
         '''write 8 byte value to given address'''
         if addr & 7:
-            raise AlignmentError()
+            self.write32(addr, data & 0xFFFFFFFF)
+            self.write32(addr + 4, data >> 32)
```

Comment:
What codepath is this used in? I'd prefer to keep the alignment fault '(at least by default) since this is usually a mistake when accessing registers, though I know Apple loves using unaligned 64-bit fields in some protocols. Can we make it a separate wrapper in ProxyUtils, a subclass, or something like that?